packages/nx-plugin/src/executors/autorun/utils.unit.test.ts

```diff
@@ -0,0 +1,94 @@
+import { MockInstance, expect, vi } from 'vitest';
+import { toNormalizedPath } from '@code-pushup/test-utils';
+import {
+  parseAutorunExecutorOnlyOptions,
+  parseAutorunExecutorOptions,
+} from './utils';
+
+describe('parseAutorunExecutorOnlyOptions', () => {
+  it('should provide NO default projectPrefix', () => {
+    expect(parseAutorunExecutorOnlyOptions({})).toStrictEqual(
+      expect.not.objectContaining({ projectPrefix: expect.anything() }),
+    );
+  });
+
+  it('should process given projectPrefix', () => {
+    expect(
+      parseAutorunExecutorOnlyOptions({ projectPrefix: 'cli' }),
+    ).toStrictEqual(expect.objectContaining({ projectPrefix: 'cli' }));
+  });
+
+  it('should provide NO default dryRun', () => {
+    expect(parseAutorunExecutorOnlyOptions({})).toStrictEqual(
+      expect.not.objectContaining({ dryRun: expect.anything() }),
+    );
+  });
+
+  it('should process given dryRun', () => {
+    expect(parseAutorunExecutorOnlyOptions({ dryRun: false })).toStrictEqual(
+      expect.objectContaining({ dryRun: false }),
+    );
+  });
+
+  it('should provide default onlyPlugins', () => {
+    expect(parseAutorunExecutorOnlyOptions({})).toStrictEqual(
+      expect.not.objectContaining({ onlyPlugins: ['json'] }),
+    );
+  });
+
+  it('should process given onlyPlugins', () => {
+    expect(
+      parseAutorunExecutorOnlyOptions({ onlyPlugins: ['md', 'json'] }),
+    ).toStrictEqual(expect.objectContaining({ onlyPlugins: ['md', 'json'] }));
+  });
+});
+
+describe('parseAutorunExecutorOptions', () => {
+  let processEnvSpy: MockInstance<[], NodeJS.ProcessEnv>;
+  beforeAll(() => {
+    processEnvSpy = vi.spyOn(process, 'env', 'get').mockReturnValue({});
+  });
+
+  afterAll(() => {
+    processEnvSpy.mockRestore();
+  });
+
+  it('should leverage other config helper to assemble the executor config', () => {
+    processEnvSpy.mockReturnValue({ outputDir: 'from-env-vars' });
```

Comment:
There's no assertion regarding this environment variable.

Reply:
updated!